eventing/README.md

```diff
@@ -18,143 +18,171 @@ The above concerns are consistent with the
-outside Kubernetes and routing them to the cluster, expressed as a Feed
-(typically, to a Channel on a Bus, but could also be directly to another
-endpoint). Right now, we only have a few generic Sources, but we plan to add
-more interesting and specific Sources over time.
-
-- **Feed** is a primitive object defining a connection between an EventType and
-  the action (as an
-  [CloudEvents compatible HTTP endpoint](https://github.com/cloudevents/spec/blob/master/http-transport-binding.md)).
-- **EventType** and **ClusterEventType** describe a specific set of events with a
-  common schema which are emitted by an EventSource. EventType is
-  namespace-scoped, while ClusterEventTypes are installed by an administrator
-  and available in all namespaces in your cluster.
-- **EventSource** and **ClusterEventSource** describe an external system which
-  may produce one or more EventTypes.
-
-We currently have 3 sources implemented:
-
-- [K8sevents](https://github.com/knative/eventing/tree/master/pkg/sources/k8sevents)
-  collects
-  [Kubernetes Events](https://kubernetes.io/docs/reference/generated/kubernetes-api/v1.10/#event-v1-core)
-  and presents them as CloudEvents.
-  ```bash
-  kubectl apply --filename https://storage.googleapis.com/knative-releases/eventing/latest/release-source-k8sevents.yaml
-  ```
-- [GitHub](https://github.com/knative/eventing/tree/master/pkg/sources/github)
-  collects pull request notifications and presents them as CloudEvents.
-  ```bash
-  kubectl apply --filename https://storage.googleapis.com/knative-releases/eventing/latest/release-source-github.yaml
-  ```
-- [GCP PubSub](https://github.com/knative/eventing/tree/master/pkg/sources/gcppubsub)
-  collects events published to a GCP PubSub topic and presents them as
-  CloudEvents.
-  ```bash
-  kubectl apply --filename https://storage.googleapis.com/knative-releases/eventing/latest/release-source-gcppubsub.yaml
-  ```
-
-### Flows
-
-Lastly, we have a higher-level abstraction called a Flow which bundles up the
-specification from the Source to the endpoint, optionally allowing you to choose
-the Channel and Bus which the event is routed over. (Otherwise, there is a
-default Bus used to provision a Channel.)
-
-- **Flow** is the top-level user-facing concept in Eventing; it describes the
-  desired path from an external Source of events to a destination that will
-  react to the events. There is only a single type of Flow, which is installed
-  by the core Knative Eventing install.
-
-## Supporting Libraries
-
-In addition to the above, there is a golang library at
-[`pkg/event`](https://github.com/knative/eventing/tree/master/pkg/event) for
-handling CloudEvents and easily creating a container which processes CloudEvents
-delivered over HTTP. You can look at several of the samples for usage examples.
+The eventing infrastructure supports two forms of event delivery at the moment:
+
+1. Direct delivery from a source to a single Service. In this case, the Source
```

Comment:
A core k8s service or a knative service ? (not trolling, honest question)

Reply:
Yes, both are supported.  Ambiguity was intentional, but if you have a better suggestion how to express this, I'm all ears.